coderdojochi/migrations/0005_auto_20160726_2010.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-07-26 20:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('coderdojochi', '0004_auto_20160718_2044'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='mentororder',
+            name='waitlist_offer_sent_at',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='mentororder',
+            name='waitlisted',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='mentororder',
+            name='waitlisted_at',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='order',
+            name='waitlist_offer_sent_at',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='order',
+            name='waitlisted',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='order',
+            name='waitlisted_at',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='equipment',
+            name='force_update_on_next_boot',
+            field=models.BooleanField(default=False, verbose_name=b'Force Update'),
+        ),
+        migrations.AlterField(
+            model_name='equipment',
+            name='last_system_update',
+            field=models.DateTimeField(blank=True, null=True, verbose_name=b'Last Update'),
+        ),
+        migrations.AlterField(
+            model_name='equipment',
```

Comment:
@karbassi these were changes someone else made that neglected to create the migrations so I'm just including them here.  they are pretty minor, i think we talked briefly about them, they support looking up choicefield labels inline which i assume we arent doing or it would be breaking :)